number_test.go

```diff
@@ -256,6 +256,44 @@ func TestNumber_InDelta(t *testing.T) {
 	value.chain.clearFailed()
 }
 
+func TestNumber_InDeltaRelative(t *testing.T) {
```

Comment:
* Could you also add a subtests that cover corner cases when n.value, value, and delta are nan, inf, and zero?
* Also please add calls to new methods to TestNumber_FailedChain.